pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -125,7 +125,6 @@ func (r *reconcilerBase) upsertServiceAccount(
 	auth configsync.AuthType,
 	email string,
 	labelMap map[string]string,
-	refs ...metav1.OwnerReference,
```

Comment:
Should the callers be modified as well as an arg is removed?

Reply:
they weren't passing any, which is why the code still compiles.

Reply:
Ack